src/routes/privacy/+page.svelte

```diff
@@ -5,90 +5,91 @@
     <li>Improve the quality of our services based on user patterns and preferences.</li>
   </ul>
 
-  <h2 id="lawful-bases">Our Lawful Bases for Processing Your Data</h2>
+  <h2 id="lawful-bases">Our lawful bases for processing your data</h2>
   <p>
     We process your data based on the consent you provide when submitting the sign-up notification form. We do not
     collect information from third parties, and all data is obtained directly from you or your device. If you do not opt
     in to receive email notifications, this policy does not apply to you. No data is sent, stored, or processed
     otherwise.
   </p>
 
-  <h2 id="retention">Data Retention Period</h2>
-  <p>
-    We retain your personal information in Google Cloud servers for up to 12 months, as outlined in the <a
-    href="#information-collected">Information We Collect</a> section. After this period,
-    your data will be permanently deleted. In most cases, your data is deleted shortly after
-    the notification email has been sent, but this is not guaranteed.
-  </p>
-
-  <h2 id="sharing">Data Sharing and Disclosure</h2>
+  <h2 id="sharing">Who we share your data with and why</h2>
   <p>
-    We may share your information with the following entities. Note that these external organizations have their own
-    privacy policies, and we are not responsible for their actions:
+    We may share your information with the following entities:
   </p>
   <ul>
-    <li><b>Google:</b> We store your data on Google Cloud servers. By accepting our privacy policy you also consent to
-      <a href="https://cloud.google.com/terms/data-processing-addendum?hl=en">Google Clouds privacy policy</a>.
+    <li><b>Google:</b> We store your data on Google Cloud's servers. See
+      <a href="https://cloud.google.com/terms/data-processing-addendum?hl=en">Google Cloud's data processing addendum</a>.
     </li>
     <li><b>Safeguarding and Legal Obligations:</b> We may share your data with government agencies or legal
-      authorities when required by law or for safeguarding reasons. This does not include private entities.
+      authorities when required by law or for safeguarding reasons.
     </li>
     <li><b>With Your Consent:</b> Your data may be shared publicly on our website, social media, or other
-      marketing channels if you have explicitly given us consent.
+      marketing channels if you have told us you consent to this.
     </li>
   </ul>
+  <p>
+    We may add or remove subprocessors at any time. They are under the same obligation as us to keep your data secure and to delete it when you or we request them to.
+  </p>
+  
+  <h2 id="retention">How long we keep your data for</h2>
+  <p>
+    We keep your personal information with our subprocesses for up to 12 months, as outlined in the <a
+    href="#information-collected">information we collect</a> section. However, we may delete your data sooner than this. After this period,
```

Comment:
Cut the we may delete sooner..., it's pointless to talk in hypotheticals in a legal document

Reply:
Ok so "at the latest within" or something like that?